storage/mysql/map_storage_test.go

```diff
@@ -15,8 +16,9 @@ func TestMapRootUpdate(t *testing.T) {
 	cleanTestDB(DB)
 	prepareTestMapDB(DB, mapID, t)
 	s := prepareTestMapStorage(DB, mapID, t)
-	tx := beginMapTx(s, t)
-	defer tx.Commit()
+	ctx := context.Background()
```

Comment:
The TODOs are because we need to fix the problem that there's no context being passed in, I think a context.Background is fine here.